tests/ml_model/test_ml_validator.py

```diff
@@ -1,6 +1,38 @@
+from credsweeper import CREDSWEEPER_DIR, ThresholdPreset
+from credsweeper.config import Config
+from credsweeper.credentials import Candidate
 from credsweeper.ml_model import MlValidator
+from credsweeper.utils import Util
 
 
 def test_ml_validator_simple_p():
-    ml_validator = MlValidator(threshold=None)
+    ml_validator = MlValidator(threshold=ThresholdPreset.medium)
     assert ml_validator is not None
+    file_name = CREDSWEEPER_DIR / "secret" / "config.json"
+    config_dict = Util.json_load(file_name)
+    config_dict["validation"] = {}
+    config_dict["validation"]["api_validation"] = False
+    config_dict["use_filters"] = True
+    config_dict["find_by_ext"] = False
+    config_dict["depth"] = 0
+    config_dict["find_by_ext_list"] = []
+    config_dict["size_limit"] = None
+    config = Config(config_dict)
+    candidate = Candidate.get_dummy_candidate(config, "test.py")
+    candidate.line_data_list[0].line = '"geheimnis" : "Jhd2gH5634"'
+    candidate.line_data_list[0].variable = "geheimnis"
+    candidate.line_data_list[0].value = "Jhd2gH5634"
+
+    decision, probability = ml_validator.validate(candidate)
+    assert 0.919577 < probability < 0.919578
+    assert decision
+
+    candidate.line_data_list[0].path = "test.yaml"
+    decision, probability = ml_validator.validate(candidate)
+    assert 0.240346 < probability < 0.240347
+    assert not decision
+
+    candidate.line_data_list[0].path = "test.py.zip"
```

Comment:
@csh519 file extension is important feature for ML decision

Reply:
I see. Then do you want to add `info` field for describe the extension?
And if the `info` field added, can ML Validation use the `info` field?

Reply:
@csh519 
Actually, info field might be only for user information. I mean, if some credentials will be found in yaml base64 encoded data with the parsing big yaml file, then no line number exists to point where the data found. But we can show path (name of field) in yaml. So, it is only for human. 
Additionally i propose to extract "file_type" in additional field. The data might be not shown in report, but used for MlValidator. The value might be different in case of original file is archive (".apk", ".docx").
"file_path" must be kept due it points to real file and changing the value we break  user logic.

Reply:
@babenek 
I totally agree with "file_path". It must be kept.
And "info" field for human and "file_type" for ML Validator are also good I think.
The idea of big yaml file is good too.

Reply:
Well, if possibly - let merge the PR, then next - I'll do changes with using `file_type`.